.goreleaser.yml

```diff
@@ -0,0 +1,32 @@
+project_name: karmada
+before:
+  hooks:
+  - tar czf crds.tar.gz -C ./charts/_crds .
+dist: bin
+env:
+- GO111MODULE=on
+builds:
+- id: kubectl-karmada
+  main: ./cmd/kubectl-karmada
+  binary: kubectl-karmada
+  env:
+  - CGO_ENABLED=0
+  ldflags:
+  - "-X github.com/karmada-io/karmada/pkg/version.gitVersion={{ .Summary }} \
+    -X github.com/karmada-io/karmada/pkg/version.gitCommit={{ .FullCommit }} \
+    -X github.com/karmada-io/karmada/pkg/version.gitTreeState=clean \
+    -X github.com/karmada-io/karmada/pkg/version.buildDate={{ .Date }}"
```

Comment:
I'm thinking what benefits we can get from the `goreleaser`?
These build configurations are already in the `Makefile`, we have to maintain a copy here.

But good to know the `goreleaser`, I'll look at the action.

Reply:
If we want to build all the binaries and other related outputs in one command, it's easy to use goreleaser, the configuration file is easy to understand, and it's easy to add new architectures, I think it's worth it.